### PR TITLE
Feat/procedure readme

### PR DESCRIPTION
**Pull request by Crash--**
Add simple readme to cozy-procedures. Since Procedures is not developer friendly yet, it's difficult to explain exactly what it does, but it's a start

### CODE REVIEW

**Comment by nono** [comment]
Ping @ptbrowne

File: packages/cozy-procedures/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<h1 align="center">Cozy Procedures</h1>
+
+<br />
+
+# What's cozy-procedures?
+
+A simple way to manipulate `io.cozy.files` adminsitrative documents.
```
Comment (ptbrowne) [inline packages/cozy-procedures/README.md:7]: administrative

File: packages/cozy-procedures/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+<br />
+
+# What's cozy-procedures?
+
+A simple way to manipulate `io.cozy.files` adminsitrative documents. 
+
+With Cozy-Procedure, you can fetch documents that match the rules you define. See [this template](./src/templates/creditApplicationTemplate.js) for an creditApplication exemple.
+
+It also helps to enhance the file you manipulate by adding automatically the corresponding metadata.
```
Comment (ptbrowne) [inline packages/cozy-procedures/README.md:11]: by adding the corresponsing metadata automatically.

File: packages/cozy-procedures/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+See https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.files_metadata/#administrative-documents for more information about administrative document.
```
Comment (ptbrowne) [inline packages/cozy-procedures/README.md:13]: s/document/documents

File: packages/cozy-procedures/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Setup
+
+### Install
+
+`npm install --save cozy-procedures`
+or
+`yarn add cozy-procedures`
```
Comment (ptbrowne) [inline packages/cozy-procedures/README.md:21]: Let's keep only `yarn` no ?

**Review by ptbrowne — APPROVED** [review]
Thanks for the README :) Some small suggestions and LGTM

**Pull request merged**